zerver/lib/test_classes.py

```diff
@@ -581,6 +581,15 @@ def send_json_payload(self, user_profile: UserProfile, url: Text,
 
         return msg
 
+    def send_json_payload_no_response(self, url: Text,
+                                      payload: Union[Text, Dict[str, Any]],
+                                      **post_params: Any) -> None:
+        old_msg = self.get_last_message()
+        result = self.client_post(url, payload, **post_params)
+        self.assert_json_success(result)
+        new_msg = self.get_last_message()
+        self.assertEqual(old_msg, new_msg)
+
```

Comment:
I don't think you need this function! In order to see if a payload was ignored or not, see [this test in our GitLab integration](https://github.com/zulip/zulip/blob/master/zerver/webhooks/gitlab/tests.py#L378).